nextflow.config

```
@@ -60,40 +89,53 @@ params {
     maps_cutoff_fdr            = 2
     maps_filter                = 'None'
     maps_model                 = 'pospoisson'
+    maps_3d_ext                = 'sig3Dinteractions.pe.txt'
     merge_map_py_source        = 'https://raw.githubusercontent.com/ijuric/MAPS/91c9c360092b25a217d91b9ea07eba5dd2ac72f4/bin/utils/genomic_features_generator/scripts/merge_map.py'
     feature_frag2bin_source    = 'https://raw.githubusercontent.com/ijuric/MAPS/91c9c360092b25a217d91b9ea07eba5dd2ac72f4/bin/utils/genomic_features_generator/scripts/feature_frag2bin.py'
     make_maps_runfile_source   = 'https://raw.githubusercontent.com/ijuric/MAPS/18c1a337f222130d7c5735d051614e2a253d5319/bin/MAPS/make_maps_runfile.py'
 
-    // Options: QC
-    skip_fastqc                = false
-    skip_plot_profile          = false
-    skip_igv                   = false
-    skip_multiqc               = false
-    skip_trackhub              = false
-    skip_enrichment            = true
+    // HiCDCPlus
+    hicdcplus_cutoff_fdr       = 0.05
+
+    // peakachu
+    peakachu_pretrained_url    = 'http://3dgenome.fsm.northwestern.edu/peakachu/HiCAR-models/HiCAR-peakachu-pretrained.'
+
+    // HiCExplorer
+    peakinteractionsthreshold  = 2
```

Given your other parmeters, shouldn't this be snake case format?

```suggestion
    peak_interactions_threshold  = 2
```

Would need to be updated elsehwere

---

fixed.